config.py

```diff
@@ -119,6 +119,11 @@ def setup_preview_dev_config():
                     "mobile": os.environ["PLATFORM_ADMIN_NUMBER"],
                 },
                 "api_key_live": os.environ["BROADCAST_SERVICE_API_KEY"],
+                "service_name": os.environ["BROADCAST_SERVICE_NAME"],
+                "service_id": os.environ["BROADCAST_SERVICE_ID"],
+                "purge_retention_limit": os.environ[
```

Comment:
I'd be tempted to call this `purge_older_than`

Reply:
done